src/robot/libraries/BuiltIn.py

```diff
@@ -618,6 +618,13 @@ def should_be_equal(self, first, second, msg=None, values=True,
         If both arguments are multiline strings, this keyword uses
         `multiline string comparison`.
 
+        If ``strip_spaces`` is given a true value (see `Boolean arguments`)
+        and both arguments are strings, the comparison is done without leading
+        and trailing spaces. If ``strip_spaces`` is given a string value
+        ``leading`` or ``trailing`` and both arguments are strings, the
+        comparison is done without leading or trailing spaces respectively.
+        The default value is ``False``.
```

Comment:
Good documentation! Few things can be still enhanced, though:

- Perhaps `leading or trailing` could be changed to `LEADING or TRAILING (case-insensitively)`. 
- The second `both arguments are strings` can be removed.
- Mentioning the actual default _value_ isn't necessary when you can see it in arguments yourself. Something like `By default spaces are not stripped.` would explain the default _behavior_ but that ought to be clear also otherwise.
- Need a note that this is new in Robot Framework 4.0 (hoping we still get this in!).

All these comments obviously apply to same doc elsewhere as well.